Tests/TuistBuildAcceptanceTests/BuildAcceptanceTests.swift

```diff
@@ -91,7 +91,8 @@ final class BuildAcceptanceTestFrameworkWithSwiftMacroIntegratedWithXcodeProjPri
     func test_framework_with_swift_macro_integrated_with_xcode_proj_primitives() async throws {
         try setUpFixture("framework_with_native_swift_macro")
         try await run(FetchCommand.self)
-        try await run(BuildCommand.self, "Framework")
+        try await run(BuildCommand.self, "Framework", "--platform", "macos")
```

Comment:
Do we build for all platforms if they're not specified?

Reply:
Currently any target that has multiple platforms will error for things like `tuist build`,  it makes sense to just build for all platforms but that work was out of scope for initial support.